migrations/2024-09-16-110154_add-unique-constraint-to-organization-name/up.sql

```diff
@@ -0,0 +1,3 @@
+-- Your SQL goes here
+ALTER TABLE organization
+ADD CONSTRAINT organization_organization_name_key UNIQUE (organization_name);
```

Comment:
how do we deal with existing duplicates?

Reply:
We need not add this constraint to the existing application. Can you move this to v2 migrations?

Reply:
done